README.md

```diff
@@ -21,21 +21,53 @@ Status
 - [ ] auth
 - [ ] load balance
 
```

Comment:
Please spell out the abbreviations. For example, async, ssl, interop, auth... Please use uppercase for the first letter of each item.